src/patternfly/components/SimpleList/simple-list.scss

```
@@ -14,7 +14,11 @@
   --#{$simple-list}__item-link--hover--Color: var(--pf-t--global--text--color--subtle);
   --#{$simple-list}__item-link--hover--BackgroundColor: var(--pf-t--global--background--color--action--plain--hover);
   --#{$simple-list}__item-link--BorderRadius: var(--pf-t--global--border--radius--tiny);
-
+  --#{$simple-list}__item-link--BorderWidth: 0;
+  --#{$simple-list}__item-link--BorderColor: var(--pf-t--global--border--color--high-contrast);
+  --#{$simple-list}__item-link--hover--BorderWidth: var(--pf-t--global--border--width--regular);
```

Since adding the new border width tokens, do you mind updating these borders to use them?

```suggestion
  --#{$simple-list}__item-link--hover--BorderWidth: var(--pf-t--global--high-contrast--border--width--regular);
```